plugins/pitchfork.py

```diff
@@ -17,7 +18,11 @@ def pitchfork(self, msg, arg):
         """
         match = re.match(r'@?([\w-]+)(?:\s+(?:down\s+)?to\s+(.+))?$',
                          arg)
-        if match:
+        labhub_util = LabHub("bot")
+        inviter = msg.frm.nick
```

Comment:
`inviter` is not a proper variable name here, give a suitable variable name for this command.